modules/_node-scripts/configuration/getProjectEntryPoints.mjs

```diff
@@ -31,5 +31,21 @@ export default function getProjectEntryPoints(projectDir = '.') {
 		entryPoints.typescript = typescript.main;
 	}
 
+	if (submodules) {
+		Object.values(submodules).forEach((submodulePath) => {
+			if (
+				!submodulePath.startsWith(
+					'src/main/resources/META-INF/resources'
+				)
+			) {
+				throw Error(
+					`❌ Submodule path '${submodulePath}' is not allowed, it must be located under 'src/main/resources/META-INF/resources/*'.`
```

Comment:
:ok_hand:

Reply:
This check looks fine to me. Now that we do check, I'm missing  a similar check with the `main` and `typescript` entry points so that we enforce this consistently.

Or else, prepend it if provided path does not start with it. But I prefer explicitness here for now, tbh